phpstan.neon

```diff
@@ -61,3 +61,7 @@ parameters:
         -
             message: '#Do not use static property#'
             path: tests/bootstrap.php
+
+        -
+            message: '#Regex must use string named capture groups instead of numeric#'
```

Comment:
I am cheating here, because I have no idea how to name it in my pattern

Reply:
I always try to come up with something that will easy my work 3 years in the future where there is a bug in regex :)

Reply:
I know there are some ?<name> things but I'm not sure where to place it O:-) especially if I'm using the whole match ($match[0]) not its parts